galsim/gsobject.py

```diff
@@ -1238,10 +1239,17 @@ def drawImage(self, image=None, nx=None, ny=None, bounds=None, scale=None, wcs=N
         @param surface_ops  A list of operators that can modify the photon array that will be
                             applied in order before accumulating the photons on the sensor.
                             [default: ()]
+        @param sensor       An optional Sensor instance, which will be used to accumulate the
```

Comment:
I think it would be worth mentioning in the drawImage docstring (above the param section somewhere) something about how the Sensor mechanics work when using FFT methods.  From the code I gather that the procedure is to first draw an image, sample photons from that image as if it were a probability distribution function, and then pass those photons to the sensor code.  (Did I get that right?)

Also, are there any combinations of `sensor != None` and `method != 'phot'` that we should warn on?  I presume only one of `sensor=SiliconSensor, method='fft'` and `sensor=SiliconSensor, method='no_pixel'` would be reasonable?